src/java.base/linux/classes/sun/nio/ch/EPollSelectorImpl.java

```diff
@@ -80,17 +79,15 @@
         this.pollArrayAddress = EPoll.allocatePollArray(NUM_EPOLLEVENTS);
 
         try {
-            long fds = IOUtil.makePipe(false);
-            this.fd0 = (int) (fds >>> 32);
-            this.fd1 = (int) fds;
+            this.eventfd = new EventFD();
         } catch (IOException ioe) {
             EPoll.freePollArray(pollArrayAddress);
             FileDispatcherImpl.closeIntFD(epfd);
             throw ioe;
         }
 
         // register one end of the socket pair for wakeups
```

Comment:
I guess the comment at line 89 no longer applies.

Reply:
You are correct, thanks.